util/predictoor/predictoor_testutil.py

```diff
@@ -0,0 +1,62 @@
+import random
```

Comment:
should this file reside here, or in predictoor/test?

Reply:
well, yes but that's because oceanutil is under util. Predictoor test util should be in the predictoor folder, in my opinion

Reply:
> > Predictoor test util should be in the predictoor folder, in my opinion
> 
> Did you mean it should be in `predicoor/test` folder?

If oceantestutil is in /util, then predictoortestutil should be in predictoor, for consistency. Let's move them later if anything else comes up